### PR TITLE
create action "EDIT project" for redux

### PR DESCRIPTION
**Pull request by tomaszprc**

### CODE REVIEW

File: src/components/ProjectElement/ProjectElement.tsx [hunk 1]
```diff
@@ -19,14 +21,81 @@ const ProjectElement = ({
...
+  const handleChangeInput = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    type: string,
+    id: number
+  ) => {
+    if (type == "title") {
+      setFormData({
+        ...formData,
+        ["id"]: id,
+        ["title"]: event.target.value,
```
Comment (korczas) [inline src/components/ProjectElement/ProjectElement.tsx:48]: brackets are not needed here

File: src/components/ProjectElement/ProjectElement.tsx [hunk 2]
```diff
@@ -19,14 +21,81 @@ const ProjectElement = ({
...
+      });
+    } else if (type == "description") {
+      setFormData({
+        ...formData,
+        ["id"]: id,
+        ["description"]: event.target.value,
```
Comment (korczas) [inline src/components/ProjectElement/ProjectElement.tsx:54]: same here

File: src/components/ProjectElement/ProjectElement.tsx [hunk 3]
```diff
@@ -19,14 +21,81 @@ const ProjectElement = ({
...
+      });
+    }
```
Comment (korczas) [inline src/components/ProjectElement/ProjectElement.tsx:56]: you can write this without if statement
```
setFormData({...formData, [type]: event.target.value})

File: src/components/ProjectElement/ProjectElement.tsx [hunk 4]
```diff
@@ -19,14 +21,81 @@ const ProjectElement = ({
...
+          <input
+            type="text"
+            onChange={(event) => handleChangeInput(event, "title", id)}
+            defaultValue={title}
+          />
+          <input
+            type="text"
+            onChange={(event) => handleChangeInput(event, "description", id)}
+            defaultValue={description}
+          />
```
Comment (korczas) [inline src/components/ProjectElement/ProjectElement.tsx:72]: why you don't use react-hook-form?

FYI: u can simplify the code
```
<input type='test' onChange={handleChangeInput} name="description" />

handleChangeInput = (e) => {setFormData({...formData, [e.currentTarget.name]: e.currentTarget.value })}
```

File: src/components/ProjectElement/ProjectElement.tsx (hunk 1 above)
Comment (tomaszprc) [reply src/components/ProjectElement/ProjectElement.tsx:48]: Merged to master, it was fixed.

File: src/components/ProjectElement/ProjectElement.tsx (hunk 2 above)
Comment (tomaszprc) [reply src/components/ProjectElement/ProjectElement.tsx:54]: Merged to master, it was fixed.

**Pull request merged**